Alignment/HIPAlignmentAlgorithm/scripts/configure.pl

```diff
@@ -2,17 +2,13 @@
 use File::Basename;
```

Comment:
@usarica it does not matter, if it is a perl script. It still has a quite generic name, i.e. please change this.
As you are anyway touching this file, this is the right moment to fix the suboptimal name.